@navikt/core/react/src/util/i18n/locales/nb.ts

```diff
@@ -44,4 +44,9 @@ export default {
       labelSuffix: "slett",
     },
   },
+  ProgressBar: {
+    progress: "{value} av {valueMax}",
```

Comment:
```suggestion
    progress: "{currentValue} av {valueMax}",
```

Reply:
What do you think about just writing "current" and "max"?

Reply:
Sounds good 👍